### PR TITLE
Explainer 1.1

### PR DESCRIPTION
**Pull request by mmccool**
Create a new Explainer for the TD 1.1 specification, a requirement for the wide review process.

The initial checkin is mostly a copy of the Explainer.md for TD 1.0, with the following changes:
- I have added a line at the top to indicate it is a WIP
- I have added a link back to the original explainer for TD 1.0
- I have broken up some long lines under Related Work 
    - so that git diffs will work better
    - so viewing and editing files in the git editor is easier
    - the MD renderer will take care of formatting and will hide these breaks
    - breaking at punctuation (at least) is a good rule - ref Knuth/TeX

To do:
- [x] Refactor material to be *just* about the TD1.1.  
    - I personally think we should factor out Architectural discussion to an Explainer under Architecture (to discuss/decide, but soon!).  
    - With four deliverables, I think it makes sense to have a single "master" explainer under Architecture and then more detailed ones under each building block.  
    - For example, the TD 1.1 explainer can include discussion of JSON-LD, relationship to OpenAPI and RAML, etc. but this material does not need to go into the Architecture explainer).
- [x] Update to new template/instructions/requirements: see https://tag.w3.org/explainers/
- [x] Update existing (remaining) content to be sure it is technically correct. 
    - For example, at one point it says the serialization is "JSON" but I wonder if this is technically true since our Content Type says "json-ld".
- [x] Explain relationship to TD 1.0, e.g defn of compatibility.
- [x] Summarize and explain changes/improvements relative to TD 1.0
   - Review high points of change log

### CODE REVIEW

**Comment by sebastiankb** [comment]
I'm not sure if we really need this since we have this here now: https://www.w3.org/WoT/documentation/
What do you think?

**Comment by sebastiankb** [comment]
I spent some time on the explainer and put this version in a more practical view (similar what I found [here](https://tag.w3.org/explainers/) at TAG).

ToDos:

- [ ] english (@mmccool can you help here)
- [ ] list of implementations 
- [ ] Related work with industry standards

Rendered version: 
https://github.com/w3c/wot-thing-description/blob/explainer11/explainer/Explainer11.md

File: explainer/Explainer11.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+## Getting started
+
+Lets assume there is a smart lamp that offers an interface with following characteristics:
+
+| |  ||  
+--- | --- |  --- |
+|Name|MyLampThing| 
+|Protocol |HTTPS| 
+|Security |Basic authentification|
```
[the remark this answers is not in the document]
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

**Comment by mmccool** [comment]
Discussion in main call:
- list of implementations -> link to web page; needs company names?  in explainer or on web page?  @egekorkan is doing an update/reorg; for Eclipse projects,  incl node-wot and ditto, list major contributors from WG)
- mccool needs to look at english, update discovery...

File: explainer/Explainer11.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
+# WoT Thing Description (TD) 1.1 Explainer (DRAFT - WIP)
+This is an updated version of the [Explainer for TD 1.0](Explainer.md).
```
Comment (mmccool) [inline explainer/Explainer11.md:2]: Since this is an explainer for the TD spec specifically, I guess I won't need to add extra material for Discovery, etc.  However, we should have an overall introduction somewhere, perhaps for Architecture, that can then refer to the individual explainers for details.  When/if we do that, though, a link back to the "general introduction" (e.g. the Architecture explainer) might be helpful, and some of the more general comments in this doc (not sure what those might be yet) could be handled there.  Although at this point, I think getting the TD explainer out the door and the wide review process going is the priority, so...

File: explainer/Explainer11.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+[//]: # (What role does the TD specification play in the WoT?)
+
+[//]: # (A brief, 4-5 paragraph explanation of the feature’s value. Outline what the feature does and how it accomplishes those goals in prose. If your feature creates UI, this is a great place to show mocks and user flows.)
+
+[//]: # (The W3C Web of Things [WoT] is intended to enable
+interoperability across IoT Platforms and application domains using Web technology.)
+
+In the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), 
+a Thing is defined as an abstraction of a physical IoT device such as a sensor (temperature, CO2, ...), an actuator (lamp, motor, ...), or a service like a weather service. The Thing Description (TD) provides descriptive metadata for a Thing's interface.
```
Comment (mmccool) [inline explainer/Explainer11.md:15]: The definition of Thing etc. have changed, make sure they are up to date.   I think these are ok, however, since you mention both physical devices (maybe "network-connected physical devices?") and services.

I would also use "network interface" instead of just "interface" since a reader might think we are describing the "user interface", etc. (which we are not).

File: explainer/Explainer11.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+TDs are designed to provide all the information needed to interact with a Thing. Specifically, it provides answers to questions like those shown in the following figure.
+
+
+![How to Interact with IoT Devices](images/explainer_td.png)
+
+
+## Getting started
+
+Lets assume there is a smart lamp that offers an interface with following characteristics:
```
Comment (mmccool) [inline explainer/Explainer11.md:30]: Let's (contraction of "Let us")

File: explainer/Explainer11.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+|Protocol |HTTPS| 
+|Security |Basic authentication| 
+|Local server address |192.168.2.134| 
+|Content type |application/json| 
+|Data / function offerings: |status (type: string) | Resource (GET): /status
+|  |toggle | Resource (POST): /toggle
+|  |overheating (type: string) | Resource (GET & longpoll): /oh
+
+
+Based on such information a Thing Description can be simple designed in the following way
```
Comment (mmccool) [inline explainer/Explainer11.md:44]: "simply" (or leave it out, "simple" is relative)

File: explainer/Explainer11.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+            "forms": [{
+                "href": "https://192.168.2.134/oh",
+                "subprotocol": "longpoll"
+            }]
+        }
+    }
+}
+```
+
+Before going in detail an important paradigm is explained that is defined by the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), namely about the interaction affordances **properties, actions**, and **events**.
```
Comment (mmccool) [inline explainer/Explainer11.md:77]: bit confusing.  As written, "in" should be "into", however I would reword completely.   Say something like

"This uses an abstraction defined by the WoT Architecture which groups interactions into three categories: properties, events, and actions."

Statements like "Before going into detail", "Note that" etc. just add more words without helping much and I would drop them.  The first sentence of a section is expected to be an introductory statement.

File: explainer/Explainer11.md [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+                "subprotocol": "longpoll"
+            }]
+        }
+    }
+}
+```
+
+Before going in detail an important paradigm is explained that is defined by the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), namely about the interaction affordances **properties, actions**, and **events**. 
+
+Each Thing and its data and functions offerings that is possible via the interface can be classified in those affordances. Sensor and/or parameter data are considered as **properties**. Functions like on/off, dimming, etc. are seen as **actions**. Data events, streams etc.  are considered as **events**.
```
Comment (mmccool) [inline explainer/Explainer11.md:79]: Each Thing's possible interactions are classified into one of these categories of affordances.

"considered as properties" -> "considered properties".
"seen as actions" -> "considered actions" (use parallel structure)
"Data events, streams, etc." -> "Notifications and data streams"are considered events"

File: explainer/Explainer11.md [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+        }
+    }
+}
+```
+
+Before going in detail an important paradigm is explained that is defined by the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), namely about the interaction affordances **properties, actions**, and **events**. 
+
+Each Thing and its data and functions offerings that is possible via the interface can be classified in those affordances. Sensor and/or parameter data are considered as **properties**. Functions like on/off, dimming, etc. are seen as **actions**. Data events, streams etc.  are considered as **events**. 
+
+In this context, the MyLampThing example above would result in declaring status a property, toggling an action, and overheating an event.
```
Comment (mmccool) [inline explainer/Explainer11.md:81]: mark up status, toggling, and overheating as code (using `single quotes` in MD)

File: explainer/Explainer11.md [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+}
+```
+
+Before going in detail an important paradigm is explained that is defined by the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), namely about the interaction affordances **properties, actions**, and **events**. 
+
+Each Thing and its data and functions offerings that is possible via the interface can be classified in those affordances. Sensor and/or parameter data are considered as **properties**. Functions like on/off, dimming, etc. are seen as **actions**. Data events, streams etc.  are considered as **events**. 
+
+In this context, the MyLampThing example above would result in declaring status a property, toggling an action, and overheating an event. 
+
+Now, by reading the above example TD,  one can obtain knowledge about the Thing named `MyLampThing`, including the following:
```
Comment (mmccool) [inline explainer/Explainer11.md:83]: by reading -> by processing
one -> a client

BTW it's not really clear who or what the "client" is, maybe this can be defined earlier.  A client may be a person reading documentation, or it may be a machine wanting to interact with the Thing.

File: explainer/Explainer11.md [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+    with a POST method at URI `https://192.168.2.134/toggle`.
+- The Thing provides one Event interaction resource with the name `overheating`.
+  - The event `overheating` can be obtained at URI 
+    `https://192.168.2.134/oh` by using HTTP with a long polling subprotocol.
+    The subprotocol clarifies which of several possible mechanisms in HTTP
+    are used to provide "push" notifications.
+  - Each message pushed by the Thing is a string value.
+
+
+## What you can do with a Thing Description (TD)?
```
Comment (mmccool) [inline explainer/Explainer11.md:102]: What can you do

File: explainer/Explainer11.md [hunk 11]
```diff
@@ -0,0 +1,287 @@
...
+
+Based on such information in a TD a WoT stack such as designed based on the [W3C Web of Things Scripting API](https://w3c.github.io/wot-scripting-api/) can simple interpret the TD content and abstract all the communication details. Developer just working with the **properties, actions**, and **events** paradigm and do not need to take care protocol specifics such as the HTTP method, the resource address, port number, etc.. The following video gives an inside about this aspect:
```
Comment (mmccool) [inline explainer/Explainer11.md:104]: Based on the information in a TD, a WoT software stack such as that described in the...

simple -> simply (or, my preference, just drop it)

What does "abstract" all the communication details?  I would be more specific and say:

"... can interpret the TD content and automatically handle all the communication details."

Comment (mmccool) [reply explainer/Explainer11.md:104]: Also, "inside" should be "insight". 

Breaking things up into separate lines would be helpful for review, BTW.

Comment (mmccool) [reply explainer/Explainer11.md:104]: "A developer working with the properties, actions, and event affordances does not need to deal with protocol specific details such as..."

File: explainer/Explainer11.md [hunk 12]
```diff
@@ -0,0 +1,287 @@
...
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/lt_P2BU8e3I/0.jpg)](http://www.youtube.com/watch?v=lt_P2BU8e3I "TD usage with a programming API")
+
+This example also shows that TDs can be used to easily onboard Things into any IoT ecosystems (e.g., IoT cloud services, edge systems, etc.). TDs helps to understand what the Things offers and how the offered data and functions can be bound to applications processes (e.g., creating service mash-ups, dashboards, etc.). The following example shows how TD are used in a node-red application:
```
Comment (mmccool) [inline explainer/Explainer11.md:108]: Remove word "easily", it's not helpful.

"ecosystems" -> "ecosystem" (use of "any" means you are selecting an example, each one of which is singular).  It might also be better to say "an" rather than "any".

"TDs helps" -> "TD helps a client" (plural subject, also verb help is clearer with an object; WHO is helped?)

"application processes"

"shows how a TD is"

Comment (mmccool) [reply explainer/Explainer11.md:108]: I think node-red is also written Node-RED usually.

File: explainer/Explainer11.md [hunk 13]
```diff
@@ -0,0 +1,287 @@
...
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/oAcYbJ6P9bU/0.jpg)](http://www.youtube.com/watch?v=oAcYbJ6P9bU "TD usage in node-red")
+
+TD also helps to mange your IoT system that typically consist of a heterogeneous device landscape based on different vendors and technologies that are used (e.g., HTTP, MQTT, Modbus, etc). Directories can be used to manage the TDs (e.g., create, search, etc). [The W3C WoT Discovery](https://w3c.github.io/wot-discovery/) provides details about a standardized how the interface of such directories can look like.
```
Comment (mmccool) [inline explainer/Explainer11.md:112]: ... manage IoT systems.  An IoT system typically consists of a heterogenous set of devices provided by different vendors and based on different technologies and protocols (e.g. HTTP, MQTT, Modbus, etc).

Comment (mmccool) [reply explainer/Explainer11.md:112]: This paragraph is also the first time that Discovery is mentioned, but the purpose of Discovery is distribution, not management of TDs.  In particular directories are designed for transient storage and include features like expiry.  They aren't the best choice for long-term archival.

If you are going to introduce Discovery here, I would say something like "The WoT Discovery specification defines mechanisms for controlled distribution and access to TDs, including searchable directories in which TDs can be dynamically registered."

File: explainer/Explainer11.md [hunk 14]
```diff
@@ -0,0 +1,287 @@
...
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/oAcYbJ6P9bU/0.jpg)](http://www.youtube.com/watch?v=oAcYbJ6P9bU "TD usage in node-red")
+
+TD also helps to mange your IoT system that typically consist of a heterogeneous device landscape based on different vendors and technologies that are used (e.g., HTTP, MQTT, Modbus, etc). Directories can be used to manage the TDs (e.g., create, search, etc). [The W3C WoT Discovery](https://w3c.github.io/wot-discovery/) provides details about a standardized how the interface of such directories can look like.  
+
+
+## What else is possible with the Thing Description specification?
+Besides the definitions of how a Thing description should be designed, there are other features provided around the TD, such as the Protocol Binding concept, TD context extensions, and Thing Model definitions.
```
Comment (mmccool) [inline explainer/Explainer11.md:116]: capitalize Thing Description here

This does not really answer the question, though.   I would change the header to "What Other Features Support the Thing Description Specification?"

Then I would reword the intro as follows:

Other features complementary to the TD, including Protocol Bindings, Context Extensions, and Thing Models, support and extend its functionality.

File: explainer/Explainer11.md [hunk 15]
```diff
@@ -0,0 +1,287 @@
...
+
+
+## What else is possible with the Thing Description specification?
+Besides the definitions of how a Thing description should be designed, there are other features provided around the TD, such as the Protocol Binding concept, TD context extensions, and Thing Model definitions. 
+
+### Protocol Binding
+Thing Description is not limited to http based interfaces as shown in the example above. In general, WoT is a protocol agnostic approach and provides a common mechanism to define how specific protocols such as MQTT, HTTP, CoAP or Modbus can be mapped to the WoT’s interaction properties-action-event abstraction within the Thing Description forms definition. More details is provide in the [Protocol Binding section](https://w3c.github.io/wot-thing-description/#protocol-bindings).
```
Comment (mmccool) [inline explainer/Explainer11.md:119]: A Thing Description

http -> HTTP

forms -> `forms`

details are

File: explainer/Explainer11.md [hunk 16]
```diff
@@ -0,0 +1,287 @@
...
+
+## What else is possible with the Thing Description specification?
+Besides the definitions of how a Thing description should be designed, there are other features provided around the TD, such as the Protocol Binding concept, TD context extensions, and Thing Model definitions. 
+
+### Protocol Binding
+Thing Description is not limited to http based interfaces as shown in the example above. In general, WoT is a protocol agnostic approach and provides a common mechanism to define how specific protocols such as MQTT, HTTP, CoAP or Modbus can be mapped to the WoT’s interaction properties-action-event abstraction within the Thing Description forms definition. More details is provide in the [Protocol Binding section](https://w3c.github.io/wot-thing-description/#protocol-bindings).  
+ 
+
+### TD Context Extensions
+Through JSON-LD serialization, the WoT Thing Description provides the ability to add context knowledge from additional namespaces. This mechanism can be used to enrich the Thing Description instances with additional (e.g., domain-specific) semantics. It can also be used to import additional Protocol Bindings or new security schemes in the future.
```
Comment (mmccool) [inline explainer/Explainer11.md:123]: context knowledge from -> vocabulary and concepts defined in

"in the future": not really needed, sounds like we need a new feature.  Maybe just delete ("... can be used to import additional Protocol Bindings or new security schemes.")

File: explainer/Explainer11.md [hunk 17]
```diff
@@ -0,0 +1,287 @@
...
+ 
+
+### TD Context Extensions
+Through JSON-LD serialization, the WoT Thing Description provides the ability to add context knowledge from additional namespaces. This mechanism can be used to enrich the Thing Description instances with additional (e.g., domain-specific) semantics. It can also be used to import additional Protocol Bindings or new security schemes in the future. 
+
+ The following TD extends the TD above by introducing a second definition in the @context to declare the prefix saref as referring to SAREF. This IoT ontology includes terms interpreted as semantic labels that can be set as values of the @type field, giving the semantics of Things and their interaction affordances. In the example below, the Thing is labelled with saref:LightSwitch, the status Property is labelled with saref:OnOffState and the toggle Action with saref:ToggleCommand.
```
Comment (mmccool) [inline explainer/Explainer11.md:125]: the prefix `saref`

Maybe also put `@context` in code-quotes, `saref:LightSwitch`, etc.

File: explainer/Explainer11.md [hunk 18]
```diff
@@ -0,0 +1,287 @@
...
+            "data": {"type": "string"},
+            "forms": [{
+                "href": "https://192.168.2.134/oh"
+            }]
+        }
+    }
+}
+```
+### Thing Model definition
+One of the main intentions of a Thing Description is to provide a Consumer with all the details necessary to successfully interact with a Thing. In some IoT application scenarios, a fully detailed Thing Description, e.g., with communication metadata is not necessary (e.g., IoT ecosystems may implicitly handle communication separately), or may not be available because a new entity has not yet been deployed (e.g., IP address is not yet known). Sometimes, also a kind of class definition is required that forces capability definitions that should be available for all created instances (e.g., large-scale production of new devices).
```
Comment (mmccool) [inline explainer/Explainer11.md:167]: this is the first place you use "Consumer", but without a definition.  I would say "client" here to be consistent, OR (much earlier) talk about kinds of clients (developers reading the spec, Consumer device/services. etc.).

Comment (mmccool) [reply explainer/Explainer11.md:167]: e.g. some IoT ecosystems may...

"forces" -> "provides"

File: explainer/Explainer11.md [hunk 19]
```diff
@@ -0,0 +1,287 @@
...
+    },
+    "events": {
+        "overheating": {
+            "description": "Lamp reaches a critical temperature (overheating)",
+            "data": {"type": "string"}
+        }
+    }
+}
+```
+Thing Model definitions are identified by the "@type": "tm:ThingModel". As the example shows, it does not provide details about a single Thing instance due to the lack of communication and security metadata. This specification presents a mechanism for deriving valid Thing Description instances from such Thing Model definitions. In addition, other design concepts are specified, including how to override, extend, and reuse existing Thing Model definitions.
```
Comment (mmccool) [inline explainer/Explainer11.md:196]: using single code-quotes here instead of double quotes

File: explainer/Explainer11.md [hunk 20]
```diff
@@ -0,0 +1,287 @@
...
+   - TDs are meant to describe existing practice, not prescribe new practices.
+
+TDs are also limited to protocols that have resources and access points that can
+be described with URLs and to payload types that have an IANA media Type.
+Structured payloads are supported but should be
+conceptually interconvertible with JSON.
+
+
+## Differentiation to Thing Description 1.0
+The main difference of this new specification from Thing Description 1.0 is that it includes the Thing Model concept (see above). This was only discussed in the Annex of the Thing Description 1.0 and was called Thing Description Template that time.
```
Comment (mmccool) [inline explainer/Explainer11.md:218]: at that time

File: explainer/Explainer11.md [hunk 21]
```diff
@@ -0,0 +1,287 @@
...
+TDs are also limited to protocols that have resources and access points that can
+be described with URLs and to payload types that have an IANA media Type.
+Structured payloads are supported but should be
+conceptually interconvertible with JSON.
+
+
+## Differentiation to Thing Description 1.0
+The main difference of this new specification from Thing Description 1.0 is that it includes the Thing Model concept (see above). This was only discussed in the Annex of the Thing Description 1.0 and was called Thing Description Template that time. 
+
+A new features is also possible to signalize that a Thing is following a specific profile such as the [W3C WoT Core Profile](https://w3c.github.io/wot-profile/).
```
Comment (mmccool) [inline explainer/Explainer11.md:220]: A new feature

to signalize -> to indicate

File: explainer/Explainer11.md [hunk 22]
```diff
@@ -0,0 +1,287 @@
...
+Structured payloads are supported but should be
+conceptually interconvertible with JSON.
+
+
+## Differentiation to Thing Description 1.0
+The main difference of this new specification from Thing Description 1.0 is that it includes the Thing Model concept (see above). This was only discussed in the Annex of the Thing Description 1.0 and was called Thing Description Template that time. 
+
+A new features is also possible to signalize that a Thing is following a specific profile such as the [W3C WoT Core Profile](https://w3c.github.io/wot-profile/). 
+
+In general, TD 1.1 specification describes a superset of the features defined in Thing Description 1.0. The new minor version update is also used to redefine some features more clearly and to add additional examples to make it easier to understand.
```
Comment (mmccool) [inline explainer/Explainer11.md:222]: define compatibility?

File: explainer/Explainer11.md [hunk 23]
```diff
@@ -0,0 +1,287 @@
...
+
+The complete change log with the new features and refinements can be found in the [change log section](https://w3c.github.io/wot-thing-description/#changes). 
+
+
+
+
+## Implementations
+
+Implementations were built by
+Bosch, Smart Things, ERCIM, Hitachi, Intel, Oracle, Panasonic, Fujitsu and Siemens (**list needs to be updated**).
```
Comment (mmccool) [inline explainer/Explainer11.md:232]: , among others.  See ... for the complete list.

**Review by mmccool — COMMENTED** [review]
There are some English changes, but also some suggested changes to wording, e.g. around Discovery.

Note: this is my own PR so I can't mark this as "Request Changes", but I am.

File: explainer/Explainer11.md (hunk 23 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

File: explainer/Explainer11.md (hunk 22 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

File: explainer/Explainer11.md (hunk 21 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

File: explainer/Explainer11.md (hunk 20 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

File: explainer/Explainer11.md (hunk 19 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

File: explainer/Explainer11.md (hunk 5 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

File: explainer/Explainer11.md (hunk 4 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

File: explainer/Explainer11.md (hunk 3 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: done

**Comment by sebastiankb** [comment]
many thanks @mmccool 

Ready to merge

File: explainer/Explainer11.md (hunk 6 above)
Comment (sebastiankb) [reply explainer/Explainer11.md]: change this to "into"

File: explainer/Explainer11.md (hunk 7 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:79]: done

File: explainer/Explainer11.md (hunk 8 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:81]: done

File: explainer/Explainer11.md (hunk 9 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:83]: done

File: explainer/Explainer11.md (hunk 11 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:104]: done

File: explainer/Explainer11.md (hunk 12 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:108]: done

File: explainer/Explainer11.md (hunk 13 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:112]: done

File: explainer/Explainer11.md (hunk 14 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:116]: done

File: explainer/Explainer11.md (hunk 15 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:119]: done

File: explainer/Explainer11.md (hunk 16 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:123]: done

File: explainer/Explainer11.md (hunk 17 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:125]: done

File: explainer/Explainer11.md (hunk 18 above)
Comment (sebastiankb) [reply explainer/Explainer11.md:167]: done

**Comment by sebastiankb** [comment]
now can be merged

**Comment by sebastiankb** [comment]
from today's TD call: decided to merge
